docs/jwst/outlier_detection/arguments.rst

```diff
@@ -1,42 +1,42 @@
 .. _outlier_detection_step_args:
 
+For more details about step arguments (including datatypes, possible values
+and defaults) see :py:obj:`jwst.outlier_detection.OutlierDetectionStep.spec`.
+
 Step Arguments for Non-IFU data
 ===============================
 The `outlier_detection` step for non-IFU data has the following optional arguments
 that control the behavior of the processing:
 
-``--weight_type`` (string, default='exptime')
-  The type of data weighting to use during resampling;
-  options are 'exptime', 'error', and 'None'.
+``--weight_type``
```

Comment:
Not 100% sure I like the idea of removing the datatype, but it's always available by following the link to the actual argument definitions in the step code.

Reply:
I agree. It would be great if the spec could contain these descriptions.